app/assets/javascripts/govuk_publishing_components/lib/cookie-functions.js

```diff
@@ -0,0 +1,65 @@
+// used by the cookie banner component
```

Comment:
What's the reason for keeping this separate from the component JS? Is something else using it?

Reply:
I made the assumption that it makes more sense to preserve this as a common lib than add it to the banner and then extract it later on when needed. The code looks very generic to be embedded in the cookie banner and there are other potential usages although we avoid storing cookies as much as possible.